packages/netlify-cms-backend-bitbucket/src/API.ts

```diff
@@ -278,21 +343,20 @@ export default class API {
       entries,
       cursor: Cursor.create({
         actions: [...(next ? ['next'] : []), ...(prev ? ['prev'] : [])],
-        meta: { index, count, pageSize, pageCount },
+        meta: { page, count, pageSize, pageCount },
         data: { links: { next, prev } },
       }),
     };
   };
 
-  listFiles = async (path: string, depth = 1) => {
+  listFiles = async (path: string, depth = 1, pagelen = 20) => {
     const node = await this.branchCommitSha(this.branch);
     const result: BitBucketSrcResult = await this.requestJSON({
       url: `${this.repoURL}/src/${node}/${path}`,
       params: {
-        // sort files by filename ascending
-        sort: '-path',
```

Comment:
This actually sorts by path descending:
https://developer.atlassian.com/bitbucket/api/2/reference/meta/filtering#query-sort